### ISSUE TITLE
Vim cursor does not move with "Go to Next Problem" - Reopening: #5746

### ISSUE THREAD

**Issue by ls1xt** [VSCodeVim/Vim#8903]
Hi, I want to re-open a bug report: Vim cursor does not move with "Go to Next Problem"  #5746 

The problem is that after "Go to Next Problem" the vim cursor is not updated accordingly. Thus any hjkl movement will take place at the old cursor position.

This was fixed several years ago but the problem seems to come back consistently.

Would be great to have this resolved!

**Comment by lucaSartore** [comment]
A possible, even not ideal solution, could be found here: https://github.com/VSCodeVim/Vim/issues/5746#issuecomment-2205307612

**Comment by erquhart** [comment]
Yep, definitely a regression. Pretty high on the frustration scale.

**Comment by erquhart** [comment]
Current workaround: This pops up every hour or so, so I hit `Developer: Restart Extension Host` when it does to fix. Makes it livable, although still pretty frustrating, until a fix is released.

**Comment by erquhart** [comment]
I switched from Copilot to Supermaven and this stopped happening. Not certain that was what did it, but that's the only big change I'm aware of. Curious if anyone else would try turning off Copilot or using something else for a bit to see if it fixes it for them as well.

**Comment by loc** [comment]
Confirming that disabling the Copilot extension fixed it for me. Also fixes a similar symptom where `Ctrl-f` search would drop the cursor back to where it started after `<Esc>`ing out of the search.

**Comment by AndruC** [comment]
> ...I hit `Developer: Restart Extension Host` in the command palette when it does to fix...

Confirmed this resolved the issue for me.

**Comment by erquhart** [comment]
Feels like a memory leak, keeps getting worse until you restart the server.

**Comment by alythobani** [comment]
> Current workaround: This pops up every hour or so, so I hit Developer: Restart Extension Host in the command palette when it does to fix. Makes it livable, although still pretty frustrating, until a fix is released.

Personally I just toggle Vim off and on (run `Vim: Toggle Vim Mode` twice, keybound to `ctrl+cmd+V`). I find that a little smoother than restarting the whole extension host (which restarts all your extensions)

And yeah I experience this issue at least a few times a day these days
